actors/builtin/storage_power/storage_power_actor.go

```diff
@@ -239,9 +239,8 @@ func (a *StoragePowerActor) OnMinerEnrollCronEvent(rt Runtime, eventEpoch abi.Ch
 	return &vmr.EmptyReturn{}
 }
 
-func (a *StoragePowerActor) ReportVerifiedConsensusFault(rt Runtime, slasheeAddr addr.Address, faultEpoch abi.ChainEpoch, faultType ConsensusFaultType) *vmr.EmptyReturn {
+func (a *StoragePowerActor) ReportConsensusFault(rt Runtime, blockHeaders []byte, slasheeAddr addr.Address, faultEpoch abi.ChainEpoch, faultType ConsensusFaultType) *vmr.EmptyReturn {
```

Comment:
```suggestion
func (a *StoragePowerActor) ReportConsensusFault(rt Runtime, header1, header2 []byte, slashee addr.Address, faultEpoch abi.ChainEpoch, faultType ConsensusFaultType) *vmr.EmptyReturn {
```

Let's accept the headers separately.